cli/info.rs

```diff
@@ -427,105 +252,89 @@ mod test {
+        .unwrap();
+    let deps = vec![ModuleInfo {
+      deps: Vec::new(),
+      name: spec_d.clone(),
+      size: 12345,
+      total_size: None,
+    }];
+    let info = ModuleInfo {
+      deps,
+      name: spec_c.clone(),
+      size: 12345,
+      total_size: Some(12345),
     };
+    let mut items = HashMap::new();
+    items.insert(
+      spec_c.clone(),
+      ModuleInfoMapItem {
+        deps: vec![spec_d.clone()],
+        size: 12345,
+      },
+    );
+    items.insert(
+      spec_d.clone(),
+      ModuleInfoMapItem {
+        deps: Vec::new(),
+        size: 12345,
+      },
+    );
+    let files = ModuleInfoMap(items);
 
-    (file, spec)
-  }
-
-  #[test]
-  fn get_resolved_file_test() {
-    let (test_file_redirect, redirect) =
-      create_mock_file("test_redirect", vec![], None);
-    let (test_file, original) =
-      create_mock_file("test", vec![], Some(redirect.clone()));
-
-    let mut graph = ModuleGraph::new();
-    graph.insert(original.to_string(), test_file);
-    graph.insert(redirect.to_string(), test_file_redirect);
-
-    let file = get_resolved_file(&graph, &original);
-
-    assert_eq!(file.specifier, redirect.to_string());
+    ModuleGraphInfo {
+      compiled: Some(PathBuf::from("/a/b/c.js")),
+      dep_count: 99,
+      file_type: MediaType::TypeScript,
+      files,
+      info,
+      local: PathBuf::from("/a/b/c.ts"),
+      map: None,
+      module: ModuleSpecifier::resolve_url_or_path(
+        "https://deno.land/x/a/b/c.ts",
+      )
+      .unwrap(),
+      total_size: 999999,
+    }
   }
 
   #[test]
-  fn dependency_count_no_redirects() {
-    let (a, aspec) = create_mock_file("a", vec![], None);
-    let (b, bspec) = create_mock_file("b", vec![aspec.clone()], None);
-    let (c, cspec) = create_mock_file("c", vec![bspec.clone()], None);
-
-    let mut graph = ModuleGraph::new();
-
-    graph.insert(aspec.to_string(), a);
-    graph.insert(bspec.to_string(), b);
-    graph.insert(cspec.to_string(), c);
-
-    let count = get_unique_dep_count(&graph);
-
-    assert_eq!(graph.len(), count);
+  fn test_module_graph_info_display() {
+    let fixture = get_fixture();
+    let actual = format!("{}", fixture);
+    assert!(actual.contains(" /a/b/c.ts"));
+    assert!(actual.contains(" 99 unique"));
+    assert!(actual.contains("(12.06KB)"));
+    assert!(actual.contains("\n\nhttps://deno.land/x/a/b/c.ts"));
   }
 
   #[test]
-  fn dependency_count_with_redirects() {
-    let (a, aspec) = create_mock_file("a", vec![], None);
-    let (b, bspec) = create_mock_file("b", vec![], Some(aspec.clone()));
-    let (c, cspec) = create_mock_file("c", vec![bspec.clone()], None);
-
-    let mut graph = ModuleGraph::new();
-
-    graph.insert(aspec.to_string(), a);
-    graph.insert(bspec.to_string(), b);
-    graph.insert(cspec.to_string(), c);
-
-    let count = get_unique_dep_count(&graph);
-
-    assert_eq!(graph.len() - 1, count);
```

Comment:
There are three tests here that got removed get_resolved_file_test, dependency_count_no_redirects, dependency_count_with_redirects. Why?

Reply:
I will take a look at those again, but they are testing behaviours that are not part of the new module graph, because of the way the graph is structure, but I will double check that the logic is covered.

Reply:
Ok, I have double checked.  `ModuleGraph2` handles re-directs in a significantly different way than the old module graph.  It depends upon the `SpecifierHandler` to mediate the redirects (which is the right layer for these to be abstracted in).  This means that `ModuleGraph2` is _unaware_ of what is going on in the background, which is perfectly fine.  I went through some older issues we had when moving to the "old" `ModuleGraph` having to deal with redirects and retested with `deno run --no-check` which uses the new `ModuleGraph2` and everything is handled properly.

There are quite a few unit tests for `ModuleGraph2`, having to deal with exotic sorts of dependencies, which is the right way to test such logic, and now `info.rs` is simply a "printout" of what is in the graph, versus having to interpret it.  That means these sorts of tests are no longer applicable.